MODULE.bazel

```diff
@@ -6,7 +6,10 @@ module(
     compatibility_level = 1,
 )
 
+# Lower-bounds for runtime dependencies.
+# Do not bump these unless rules_js requires a newer version to function.
```

Comment:
As an aside, I'd also advocate for minimum version bumps if there are critical bug fixes or notable performance improvements to would improve function.